src/ig-template/IgtGame.ts

```diff
@@ -96,26 +99,21 @@ export abstract class IgtGame {
   }
 
   /**
-   * Force update all features for testing purposes
+   * For testing purposes:
+   * Force updates all features
+   * Sends the onTick event
    */
   public forceUpdate(delta: number): void {
```

Comment:
Maybe we can rename this to `fakeTick` to better reflect what it does

Reply:
Done!